libnmstate/nm/wired.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate import nmclient
+
+DEFAULT_MTU = 1500
+
+
+def create_setting(iface_state, base_con_profile):
+    mtu = iface_state.get("mtu")
+    if not mtu:
+        return None
+
+    if base_con_profile:
+        wired_setting = base_con_profile.get_setting_wired()
```

Comment:
This is not how it works with the connection-settings, see: duplicate_settings()
I remember it was a recommendation from thaller.
Could you please re-check this?

Reply:
@thom311 does this need to be changed somehow?